sekizai/helpers.py

```diff
@@ -34,6 +27,14 @@ def is_variable_extend_node(node):
             return True
     return False
 
+def get_context():
+    if engines is not None:
+        context = Context()
+        context.template = Template('')
+        return context
+    else:
+        return {}
```

Comment:
In older versions this should not matter anyway, and it's the same logic as https://github.com/ojii/django-sekizai/blob/master/sekizai/helpers.py#L18, but I can test how it behaves returning a Context